crates/ursa-gateway/src/server/route/api/v1/get.rs

```diff
@@ -88,7 +122,7 @@ pub async fn get_block_handler(
     // 1. maintain N workers keep track of indexing data
     // 2. cherry-pick closest node
     // 3. cache TTL
-    (StatusCode::OK, Json(json!(indexer_response)))
+    (StatusCode::OK, Json(json!({ "data": resp })))
```

Comment:
can we improve it a bit? how about pipelining the response without loading it to the memory?

Reply:
Talked to Scott about this and we will optimize as part of a separate pr.